### DISCUSSION TITLE
Is there a changelog available for "Windows Subsystem for Linux Preview"?

### DISCUSSION THREAD

**Discussion by trallnag** [microsoft/WSL#8771] [Q&A]
See above. Would be awesome to know what changes are included in these updates.

![image](https://user-images.githubusercontent.com/24834206/186741989-4ad3889b-77b8-4be9-84f9-5b9e270c637d.png)

**Comment by Biswa96** [comment]
The release page may help https://github.com/microsoft/WSL/releases
